cincinnati/src/plugins/mod.rs

```diff
@@ -61,15 +61,15 @@ where
 
 /// Trait to be implemented by internal plugins with their native IO type
 pub trait InternalPlugin {
-    fn run_internal(&self, InternalIO) -> Fallible<InternalIO>;
+    fn run_internal(&self, _: InternalIO) -> Fallible<InternalIO>;
```

Comment:
To me the `_` is a bit confusing as in other places it means you can't use the value. Let's give it a name to avoid any confusion.

```suggestion
    fn run_internal(&self, input: InternalIO) -> Fallible<InternalIO>;
```